src/components/template/card-type-pick-bottom-sheet/card-type-pick-bottom-sheet.tsx

```diff
@@ -0,0 +1,71 @@
+import { BottomSheet, BottomSheetContentProps } from '@/components/molecule/bottom-sheet'
+import * as styles from './card-type-pick-bottom-sheet.css'
+import { Text, Flex, FlexProps } from '@/components'
+import { CardType } from '@/types/card'
+import { IconCheck } from '@tabler/icons-react'
+import { vars } from '@/styles'
+
+export interface CardTypePickBottomSheetProps {
+  onClose: BottomSheetContentProps['onClose']
+  cardTypeList: CardType[]
+  selectedCardType?: CardType
+  onSelectCardType: (value: CardType) => void
+}
+
+export const CardTypePickBottomSheet = ({
+  cardTypeList,
+  onClose,
+  selectedCardType,
+  onSelectCardType,
+}: CardTypePickBottomSheetProps) => {
+  const handleClickCardTypeItem = (cardType: CardType) => () => {
+    onSelectCardType?.(cardType)
+    onClose()
+  }
+
+  return (
+    <BottomSheet.Root>
+      <BottomSheet.Dimmer />
+      <BottomSheet.Content className={styles.content} onClose={onClose}>
+        {cardTypeList.map(({ id, name, color }, idx) => (
+          <CardTypeItem
+            key={id}
+            name={name}
+            isSelectedCardType={selectedCardType?.id === id}
+            cardTypeColor={color}
+            onClick={handleClickCardTypeItem(cardTypeList[idx])}
```

Comment:
a: map에서 arg 분해할당하지 않으면 배열과 인덱스로 참조하지 않아도 되네요. 배열과 인덱스로 참조하신 이유가 있으신가요?
```suggestion
            onClick={handleClickCardTypeItem(cardType)}
```

Reply:
말씀하신대로 분해할당을 안쪽에 하도록 수정했습니다 확인감사합니다

[커밋](https://github.com/next-step/react-payments/pull/142/commits/066c5f2d3ce554a20deedef4f1b9a55d7c5ee0bd)